dotcom-rendering/src/model/enhanceCards.ts

```diff
@@ -75,6 +77,38 @@ const enhanceSupportingContent = (
 	});
 };
 
+const decideAvatarUrl = (
```

Comment:
This function uses the new `FETagType` type (which is the data we get from FE) and transforms the tags array into `TagType[]`. We need this transformation to be able to pass `tags` to the `getSoleContributor` function to decide if an avatar should show or not

Reply:
Can you require that tags are defined?

Reply:
(This is where I’ve added the comment)

I was thinking that `tags` should be required for this, or at lease default to an empty array.

Reply:
Yep, now done 👍